src/i965_drv_video.c

```diff
@@ -974,12 +974,18 @@ i965_GetConfigAttributes(VADriverContextP ctx,
                     attrib_list[i].value |= VA_RC_CBR;
 
                 if (profile == VAProfileVP8Version0_3 ||
-                    profile == VAProfileVP9Profile0 ||
-                    profile == VAProfileH264ConstrainedBaseline ||
+                    profile == VAProfileVP9Profile0)
+                    attrib_list[i].value |= VA_RC_VBR;
+
+                if (profile == VAProfileH264ConstrainedBaseline ||
                     profile == VAProfileH264Main ||
                     profile == VAProfileH264High ||
-                    profile == VAProfileHEVCMain)
-                    attrib_list[i].value |= VA_RC_VBR;
+                    profile == VAProfileHEVCMain) {
+                    if (IS_GEN9(i965->intel.device_info)) {
+                        attrib_list[i].value |= VA_RC_VBR;
+                        attrib_list[i].value |= VA_RC_MB;
```

Comment:
Can we declare the RC mode capacity in i965_device_info.c , just like lp_h264_brc_mode? It will make user have a unified interface to check codec capacity.